### PR TITLE
Handle newrelic in fabric

### PR DESCRIPTION
**Pull request by kinnou02**
Previously it was done manually :(

### CODE REVIEW

**Comment by nonifier** [comment]
Out of curiosity, where can we find the dependency to Newrelic's Python Agent ? Couldn't find it in any `requirement.txt`

**Comment by kinnou02** [comment]
Good point, forgot to add it!

File: fabfile/component/jormungandr.py [hunk 1]
```diff
@@ -123,6 +123,8 @@ def upgrade_ws_packages():
 
     # We want the version of the system for these packages
     run('''sed -e "/protobuf/d" -e "/psycopg2/d"  /usr/share/jormungandr/requirements.txt > /tmp/jormungandr_requirements.txt''')
+    #add newrelic as a dependency
+    run('''echo 'newrelic==2.70.0.51' >> /tmp/jormungandr_requirements.txt''')
```
Comment (nonifier) [inline fabfile/component/jormungandr.py]: Did you had a look at Distributed Tracing in newrelic ? Looks interesting for our distributed scenario, but it's only available from v.4.2.

Comment (kinnou02) [reply fabfile/component/jormungandr.py]: I took the version used on production, and will latter upgrade to the 4.2 after testing a bit in dev

File: templates/jormungandr/newrelic.ini.jinja [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+# proxy_user =
+# proxy_pass =
+
+# Capturing request parameters is off by default. To enable the
+# capturing of request parameters, first ensure that the setting
+# "attributes.enabled" is set to "true" (the default value), and
+# then add "request.parameters.*" to the "attributes.include"
+# setting. For details about attributes configuration, please
+# consult the documentation.
+# attributes.include = request.parameters.*
```
Comment (nonifier) [inline templates/jormungandr/newrelic.ini.jinja]: After playing a bit with it, I wonder if capturing the request parameters would be interesting to understand how people are using the API, like options always/never used. What's you thoughts on that ?

Comment (kinnou02) [reply templates/jormungandr/newrelic.ini.jinja]: We already have them in newrelic, it's probably overridden by the "server-side" configuration.
This is the file currently used in production, it's a bit old and we should update it when upgrading the version of newrelic we use

**Pull request merged**